atlp-rwanda/cabal-phantom-frontend
# Setup Github Action

### **What about this PR?**
This PR has github action setup

### **Description of Task to be completed**
- Setup continous integration with github action

### **How should be tested?**
- View the  [CI Badge](https://github.com/atlp-rwanda/cabal-phantom-frontend/blob/ch-setup-github-action/Readme.md)
- Check this repo action through [here](https://github.com/atlp-rwanda/cabal-phantom-frontend/actions)
 
### **Relevant pivots tracker**
No pivot tracker.
